docs/Getting Started/Install and Run/index.md

```diff
@@ -59,11 +59,12 @@ CMake is used to configure your SDL Core build before you compile the project, t
 
 After installing the appropriate dependencies for your build configuration, you can run `cmake` with your chosen options. 
 
-Begin by creating a build folder outside of SDL Core source folder, for example:
+Begin by creating a build folder **outside** of SDL Core source folder, for example:
 
 ```bash
-mkdir ../sdl_build
-cd ../sdl_build
+cd ..
+mkdir build
+cd build
```

Comment:
```suggestion
cd ..
mkdir sdl_build
cd sdl_build
```
Minor detail, but I do think the existing naming is less likely to cause conflicts with other existing folders in someone's system since it is a bit more specific. In practice it doesn't matter too much though

Reply:
Agreed.  I just found when running cMake and trying to use sdl_build never worked for me but using build does everytime...I don't know if that's because programmatically this name is hardcoded somewhere or what...but trying to use it when running through the steps always caused problems.